client/src/pages/about/[slug].js

```diff
@@ -15,21 +16,19 @@ const CoworkerPage = ({ data, preview = false }) => {
     enabled: preview,
   })
   const post = filterDataToSingleItem(previewData, preview)
+  const customMediaType = { image: { ...post.heroImage }, type: 'image' }
 
   return (
     <PageWrapper footerDark>
       {preview && <ExitPreviewLink />}
       {post && (
-        <>
-          <CoworkerHero
-            title={post?.fullname ?? post.fullname}
-            heroImage={post?.heroImage ?? post.heroImage}
-          >
-            {post?.role ?? post.role}
-          </CoworkerHero>
-          <Medarbetare info={post ?? post} />
-        </>
+        <Hero
+          title={post.fullname && post.fullname}
+          subtitle={post.role && post.role}
+          mediaType={post.heroImage && customMediaType}
+        />
```

Comment:
```suggestion
        <Hero
          title={post.fullname}
          subtitle={post.role}
          mediaType={post.heroImage && customMediaType}
        />
```
`post.fullname && post.fullname` will reteurn the same value as `post.fullname`

Reply:
Hmm yes, these existed from before but I have been unsure about it. I'll move the checks to inside the component.